components/CTA.tsx

```diff
@@ -17,6 +17,19 @@ const Icon = ({ path }: { path: string }) => (
   </button>
 );
 
+const CompletedIcon = ({ path }: { path: string }) => (
```

Comment:
Could you extract it to a new file and add confetti animation from `AnimatedCheckIcon` ?

Then we can delete `AnimatedCheckIcon` :)